Towny/src/main/java/com/palmergames/bukkit/towny/TownyMessaging.java

```diff
@@ -668,15 +672,15 @@ public static void sendJailList(CommandSender sender, Town town, int page, int t
 		} else {
 			jailsFormatted = new TextComponent[10];
 		}
-		String headerMsg = ChatColor.GOLD + "# " +
-							ChatColor.DARK_GRAY + "- "+
-							ChatColor.GREEN + "Jail Name " +
-							ChatColor.DARK_GRAY + "- "+
-							ChatColor.BLUE + "Coord " +
-							ChatColor.DARK_GRAY + "- " +
-							ChatColor.YELLOW + "Cell Count " +
-							ChatColor.DARK_GRAY + "- " +
-							ChatColor.RED + "Primary Jail";
+		String headerMsg = NamedTextColor.GOLD + "# " +

```

Comment:
The implementation of NamedTextColor#toString is just the name, so this ends up being the string "gold# " (with no color). To convert this you'd need to use minimessage or use a component from the get to.